extension/script/docs-rs.js

```diff
@@ -116,8 +116,9 @@ document.addEventListener("DOMContentLoaded", async () => {
 async function enhanceFeatureFlagsMenu(menu) {
     // Use rawCrateName to fetch the Cargo.toml, otherwise will get 404.
     let cargoTomUrl = `https://docs.rs/crate/${rawCrateName}/${crateVersion}/source/Cargo.toml`;
-    let response = await fetch(cargoTomUrl);
-    let content = await response.text();
+    let crateAPIURL = `https://crates.io/api/v1/crates/${rawCrateName}/${crateVersion}`;
```

Comment:
The `crateVersion` could be `latest` which is invalid for the API of crates.io. For example: https://crates.io/api/v1/crates/tokio/latest

We should ensure the `enhanceFeatureFlagsMenu()` function is called after `crateVersion = parseCrateVersionFromDOM()`, see line 97.